yellowbrick/regressor/influence.py

```diff
@@ -0,0 +1,280 @@
+# yellowbrick.regressor.influence
+# Visualize the influence and leverage of individual instances on a regression model.
+#
+# Author:   Benjamin Bengfort <benjamin@bengfort.com>
+# Created:  Sun Jun 09 15:21:17 2019 -0400
+#
+# Copyright (C) 2019 The scikit-yb developers
+# For license information, see LICENSE.txt
+#
+# ID: influence.py [] benjamin@bengfort.com $
+
+"""
+Visualize the influence and leverage of individual instances on a regression model.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import scipy as sp
+
+from yellowbrick.base import Visualizer
+from sklearn.linear_model import LinearRegression
+
+
+##########################################################################
+## Cook's Distance
+##########################################################################
+
+
+class CooksDistance(Visualizer):
+    """
+    Cook's Distance is a measure of how influential an instance is to the computation of
+    a regression, e.g. if the instance is removed would the estimated coeficients of the
+    underlying model be substantially changed? Because of this, Cook's Distance is
+    generally used to detect outliers in standard, OLS regression. In fact, a general
+    rule of thumb is that D(i) > 4/n is a good threshold for determining highly
+    influential points as outliers and this visualizer can report the percentage of data
+    that is above that threshold.
+
+    This implementation of Cook's Distance assumes Ordinary Least Squares regression,
+    and therefore embeds a ``sklearn.linear_model.LinearRegression`` under the hood.
+    Distance is computed via the non-whitened leverage of the projection matrix,
+    computed inside of ``fit()``. The results of this visualizer are therefore similar
+    to, but not as advanced, as a similar computation using statsmodels. Computing the
+    influence for other regression models requires leave one out validation and can be
+    expensive to compute.
+
+    .. seealso::
+        For a longer discussion on detecting outliers in regression and computing
+        leverage and influence, see `linear regression in python, outliers/leverage
+        detect <http://bit.ly/2If2fga>`_ by Huiming Song.
```

Comment:
I prefer these `.. seealso::` blocks for links - I created the bit.ly link in my account to minimize the amount of text required.